### DISCUSSION TITLE
Console is not printing anything

### DISCUSSION THREAD

**Discussion by suh-ana** [Jay-study-nildana/FrontEndForStudents#46] [Coding Questions]
Hi Jay, 

Hope you are doing well.

Actually I was learning JS, I was trying to print simple hello in VS code but it kept throwing 404 not found error. Dropping the screenshot. Please help me out. I am unable to find the solution.

![Screenshot (203)](https://github.com/Jay-study-nildana/TutoringTrainingStudents/assets/67051200/6e0092c4-ecd5-4df5-aa8a-e31d3b45f636)

Thanks

**Comment by Jay-study-nildana** [comment]
@suh-ana 

**Part One:** 

Thank you for posting your question. In the future, please post questions in this link, 

https://github.com/Jay-study-nildana/FrontEndForStudents/discussions 

**Part Two:**

Now, when you are asking doubts, you should share the code. look at this link on how to ask doubts and questions.

read this link to understand how to ask questions: https://github.com/Jay-study-nildana/FrontEndForStudents/discussions/25 

**Part Three**

For beginners, I have made some videos, which show basic things (like console logging and things)

https://www.youtube.com/playlist?list=PL7SHzFbuSdW0-VhFwVTD2CRzpxkHnwKXm
https://www.youtube.com/playlist?list=PL7SHzFbuSdW3xtOOl8nb5AGAaZNR3smel

**Part Four**

Now, based on your screenshot, I think, you are not using console log, or, you have not linked the javascript file.

Look at this folder, on how to get console log working, https://github.com/Jay-study-nildana/FrontEndForStudents/tree/main/JSForStudents/Bootstrap5TemplateStarterAmeeshaPatel

You can also look at this codepen. 

https://codepen.io/jay-pancodu/pen/abjbrxK

If you are new to codepen, watch this video. 

https://www.youtube.com/watch?v=aVqe0T0dNfg

**Finally,**

Remember, if you are new to coding, https://www.youtube.com/watch?v=BjBa8mzyBg0

If you have more questions, please feel free to ask questions in this discussion forum. You are already following me on social media, so, feel free to email me if you have doubts. 

All the best for your developer journey. 

Jay (Vijayasimha BR)
